From 6f3f61023e36a07d23dacee05a46f63e7cdad58b Mon Sep 17 00:00:00 2001
From: Murillo Arantes <murillo.arantes@windriver.com>
Date: Wed, 29 Oct 2025 09:20:40 -0300
Subject: [PATCH] STX: Adjust libc6-dev version compatibility

To ensure compatibility, this patch adjusts the build dependency of
libvirt to use libc6-dev version 2.31-13 instead of 2.31-14. This
modification was necessary to align the libc6-dev version with the one
currently used on the platform. Using libc6-dev version 2.31-14 would
break the build of several other packages that directly depend on
version 2.31-13 and could introduce the risk of side effects on the
system.

Signed-off-by: Murillo Arantes <murillo.arantes@windriver.com>
---
 debian/control | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control b/debian/control
index 1d3c576fb..0d2f44c77 100644
--- a/debian/control
+++ b/debian/control
@@ -18,7 +18,7 @@ Build-Depends:
  libattr1-dev [linux-any],
  libaudit-dev [linux-any],
  libblkid-dev,
- libc6-dev (>= 2.31-14~),
+ libc6-dev (>= 2.31-13~),
  libcap-ng-dev [linux-any],
  libcurl4-gnutls-dev,
  libdevmapper-dev [linux-any],
-- 
2.34.1

